### DISCUSSION TITLE
How to modify the hero-name width, so that longer names are not splitted

### DISCUSSION THREAD

**Discussion by lazarusA** [vuejs/vitepress#3381] [Q&A] [answered]
The current behaviour splits the hero-name into two columns, and apparently of equal size. So, some times we could have longer hero-names that would not fit into those boxes. Having an awkward split. 

![Screenshot 2023-12-26 at 12 13 56](https://github.com/vuejs/vitepress/assets/19525261/a8bc187b-8dbf-45a0-a6b3-868bf892e6b6)

Possible approaches? maybe.. options to reduce the font size, or extend the text-width or bounding box?

**Comment by brc-dd** [comment, marked as answer]
You can change that stuff directly from css. Something like this should work:

```css
.VPHero .clip {
  white-space: pre;
  font-size: ...;
}
```

**Reply by lazarusA** [reply]
Thanks, still getting use on how to set all this attributes via  css. There are so many options, but not many examples. Thanks again.
